shelley/chain-and-ledger/executable-spec/test/Generator/Delegation.hs

```diff
@@ -13,18 +13,20 @@ module Generator.Delegation
 import           Data.Sequence (Seq)
 import qualified Data.Sequence as Seq
 import           Hedgehog (Gen)
+import           Lens.Micro ((^.))
 
 import qualified Hedgehog.Gen as Gen
 
 import           Coin (Coin (..))
-import           Delegation.Certificates (pattern RegKey)
+import           Delegation.Certificates (pattern DeRegKey, pattern RegKey, decayKey)
 import           Examples (unsafeMkUnitInterval)
 import           Generator.Core (toCred)
-import           Ledger.Core (dom, (∉))
-import           LedgerState (_dstate, _pstate, _stKeys, _stPools)
+import           Ledger.Core (dom, (∈), (∉))
+import           LedgerState (dstate, keyRefund, stKeys, _dstate, _pstate, _stKeys, _stPools)
 import           MockTypes (DCert, DPState, DState, KeyPair, KeyPairs)
 import           PParams (PParams (..), emptyPParams)
-import           Slot (Epoch (..))
+import           Slot (Epoch (..), Slot)
```

Comment:
Would it make sense to change the implicit exports in this module into explicit ones? It is much easier to find out when symbols came from, specially when doing reviews on github.

Reply:
Do you mean replacing e.g. `Foo(..)` with `Foo(bar, baz)` in the import lists? I think that's reasonable!

Reply:
Yes :) We have a bunch of `import M (X (...))` in this module (and in our codebase). Maybe we can replace them little by little...

Reply:
Cool, good idea. I’ll make that change here.